krabs/krabs/ut.hpp

```diff
@@ -19,6 +19,14 @@ namespace krabs { namespace details {
     struct ut {
 
         typedef krabs::provider<> provider_type;
+		
+		struct FilterSettings
+		{
+			std::vector<unsigned short> m_OrigEventIds;
+			std::tuple<UCHAR, ULONGLONG, ULONGLONG, UCHAR> flagsTuple;
+		};
+
+		using ProvidersFilterSettings = std::map<krabs::guid, FilterSettings>;
```

Comment:
nit: `ProviderFilterSettings`

Reply:
Please use the `typedef` style for this instead of `using` since we don't use `using` elsewhere:

```suggestion
		typedef std::map<krabs::guid, FilterSettings> provider_filter_settings;
```

Reply:
Done